### PR TITLE
Lesson 4 1

### PR DESCRIPTION
**Pull request by Tugeeta**
Added all the sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,20 +1,219 @@
...
+            <p> 2009-2010: <br> Worked as a junior level developer, Broadgate Inc, Troy, MI</p>
+        </li>
+          </ul>
+        </div>
+ </section>
+ &nbsp &nbsp
+ <section>
+ <h1 class="h1-section">Connect</hi>
+  <h3> Email: gtturumella@gmail.com &nbsp GitHub
+  <a href="https://github.com/Tugeeta"><img src="github.jpg" width="15"> &nbsp &nbsp Linkedin</a>
```
Comment (mccalluc) [inline index.html:214]: character entities should have a semi-colon: `&nbsp;`... but if you need to change the padding, probably better to do that with css.

File: index.html [hunk 2]
```diff
@@ -1,20 +1,219 @@
...
+            <p>I have been out of work for the past 12 years as I was taking care of home and my son.  I have been doing some volunteering work in my son’s school as well as chaperoning the field trips before pandemic.  I am actively managing home, looking after my son as well as self teaching myself on current trends in web developer job market.</p><br><br>
+        </li>
+    
+            <li>
+            <p> 2009-2010: <br> Worked as a junior level developer, Broadgate Inc, Troy, MI</p>
+        </li>
+          </ul>
+        </div>
+ </section>
+ &nbsp &nbsp
```
Comment (mccalluc) [inline index.html:210]: not needed.

**Review by mccalluc — CHANGES_REQUESTED** [review]
GDlogo.jpg shouldn't be checked in. For the other logos, PNGs are usually preferred to JPG for line art.

**Comment by Tugeeta** [comment]
Good Afternoon

Thanks for reviewing and I noted the changes.  I will apply to it.

Thanks and Regards!
Have a nice day!
Geeta

**Pull request merged**